pyproject.toml

```diff
@@ -1,18 +1,20 @@
-# Copyright (c) 2023, NVIDIA CORPORATION.
+# Copyright (c) 2023-2025, NVIDIA CORPORATION.
 
 [build-system]
 requires = [
     "hatch-jupyter-builder>=0.5",
     "hatch-nodejs-version>=0.3.2",
-    "hatchling>=1.5.0",
+    "hatchling>=1.27.0",
```

Comment:
Hatch 1.27.0 added support for license files with PR: https://github.com/pypa/hatch/pull/1829

Hence we bump the minimum version accordingly